VideoLocker/src/main/java/org/edx/mobile/base/BaseSingleFragmentActivity.java

```diff
@@ -131,24 +131,16 @@ public void finishProcess(){
         hideLoadingProgress();
     }
 
-    public void onMessage(@NonNull MessageType messageType, @NonNull String message){
+    public void onMessage(@NonNull MessageType messageType, @NonNull String message) {
         //TODO - -we need to define different UI message view for different message type?
-        switch ( messageType ){
+        switch (messageType) {
             case FLYIN_ERROR:
```

Comment:
Do we need to start deprecating some of these cases if we are going to just combine them?